src/main/rx/rx.c

```diff
@@ -79,6 +79,7 @@ static pt1Filter_t frameErrFilter;
 
 #ifdef USE_RX_LINK_QUALITY_INFO
 static uint16_t linkQuality = 0;
+static uint8_t rf_mode = 0;
```

Comment:
Probably `rfMode` is better to be consistent with `linkQuality`.

Reply:
no problem